apps/OpenSpace/ext/launcher/src/launcherwindow.cpp

```diff
@@ -637,20 +678,117 @@ void LauncherWindow::openProfileEditor(const std::string& profile, bool isUserPr
     }
 }
 
-void LauncherWindow::openWindowEditor() {
-    SgctEdit editor(this, _userConfigPath);
-    int ret = editor.exec();
-    if (ret == QDialog::DialogCode::Accepted) {
-        sgct::config::Cluster cluster = editor.cluster();
+void LauncherWindow::editRefusalDialog(const std::string title, const std::string msg) {
+    QMessageBox msgBox;
+    msgBox.setText(QString::fromUtf8(msg.c_str()));
```

Comment:
Better to:
`QString::fromStdString(msg);` and `QString::fromStdString(title)`.  Our `title` and `msg` are not Utf8 anyway